databases/week2/queries.sql

```diff
@@ -0,0 +1,173 @@
+--Part 1: Working with tasks
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+  task(
```

Comment:
All these newlines are unnecessary and decrease readability.

Reply:
It's due the formatter that's installed- even I do not like this formatting. But I will keep in mind and try to not repeat it next time.